locales/en-US/app.l20n

```diff
@@ -73,6 +74,17 @@
 <feedbackUninstallAnswerNotUseful "This isn't useful for me.">
 <feedbackUninstallAnswerOther "Something else.">
 
+<retireDialogTitle "Retire from Test Pilot?">
+<retireMessage "If you retire from Test Pilot, we will disable any active tests, uninstall the Test Pilot add-on, remove your account information from our servers, and generally just leave you alone.">
+<retireSubmitButton "Retire">
+<retireCancelButton "cancel">
```

Comment:
Lowercase is intended, I think. A least, [it was lowercase in the prototype](http://people.mozilla.org/~jgruen/test-pilot/home.html). Also, the feedback dialog is getting pulled out in #567, so we can correct that discrepancy via deleting it.